EIPS/eip-1679.md

```diff
@@ -34,6 +34,8 @@ This meta-EIP specifies the changes included in the Ethereum hardfork named Ista
     [pending audit](https://medium.com/ethereum-cat-herders/progpow-audit-goals-expectations-75bb902a1f01),
     above and beyond standard security considerations, that should be evaluated
     prior to inclusion.
+- [EIP-1108](https://eips.ethereum.org/EIPS/eip-1108): Reduce alt_bn128 precompile gas costs  
+  - There is currently an outstanding [pull request](https://github.com/ethereum/EIPs/pull/1987) that updates the gas costs to the latest benchmarks
```

Comment:
I'd prefer we merge #1987 first, remove this comment and merge this to avoid the extra PRs needed to remove this line.

Reply:
Just merged that guy, so it looks like @zac-williamson you just need to drop the note on line 38 and we're maybe good to go here?

Reply:
@zac-williamson can you please do that? I'd like to merge this.